src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -949,6 +949,13 @@ public class BoolToCanvasCursorConverter : IValueConverter
         }
     }
 
+    /// <summary>
+    /// Obsolete class of ShowHideConsoleMenuItemConverter
+    /// Using of this class will produce compile warnings
+    /// TODO: To be removed in Dynamo 2.0
+    /// TODO: Resource String DynamoViewViewMenuHideConsole to be removed in Dynamo 2.0
+    /// </summary>
+    [System.Obsolete("This class is obsolete.")]
     public class ShowHideConsoleMenuItemConverter : IValueConverter
```

Comment:
This converter will be obsolete because we no longer need this public converter class to return different string for console display option.